app/static/styles/map.css

```diff
@@ -0,0 +1,35 @@
+
```

Comment:
change it to map.scss so that we can use some common sass variables to style it. You can read up on it here (http://sass-lang.com/guide)